ydb/core/kqp/ut/view/input/cases/count_episodes/create_view.sql

```diff
@@ -0,0 +1,9 @@
+CREATE VIEW `/Root/count_episodes` WITH (security_invoker = TRUE) AS
```

Comment:
What do these tests check? Do you have a validation for the results of these queries?

Reply:
This particular test (`count_episodes`) is a way to test `COUNT` inside a view, but with a bit more complex query than just `SELECT COUNT(*) FROM some_table`

Reply:
It's an intresting approach. Thank you for exlanation :)